act/plotting/distributiondisplay.py

```diff
@@ -863,3 +863,83 @@ def plot_violin(
                 self.axes[subplot_index].set_yticks([])
 
         return self.axes[subplot_index]
+
+    def plot_pie_chart(
+        self,
+        fields,
+        dsname=None,
+        subplot_index=(0,),
+        set_title=None,
+        autopct='%1.1f%%',
+        percent_kwargs=None,
+        **kwargs,
+    ):
+        """
+        This procedure will produce a pie chart for the selected fields.
+
+        Parameters
+        ----------
+        fields : list
+            The list of fields to calculate percentages on for the pie chart.
+        dsname : str or None
+            The name of the datastream the field is contained in. Set
+            to None to let ACT automatically determine this.
+        subplot_index : tuple
+            The subplot index to place the plot in
+        set_title : str
+            The title of the plot.
+        autopct : str
+            Format string for the percentages. Default is float with one
+            decimal place. If this parameter is set to None, no percentage
+            string values are displayed.
+        percent_kwargs : dict
+            Dictionary of parameters to be used in the act.utils.calculate_percentages
+            function. None will use default parameters.
+
```

Comment:
@zssherman do we need to list kwargs down here in the parameters list?

Reply:
@zssherman My view on it would be to keep it, just because we are using two kwargs, one for ax.pie and the other for calculate percentages

Reply:
Sorry, I was just referring to the documentation and just adding 2 lines to list the kwargs as a parameter.  I didn't see it listed in the documentation.  

@mgrover1 you're noting that for the percent_kwargs right and not for the kwargs passed to the matplotlib pie function right?

Reply:
@AdamTheisen There is a mention below that shows it goes to ax.pie. And yeah Max and I discussed he was thinking for verbose, to add the parameter from calculated_percentages as there is not many and it easier to read and access.

Reply:
@AdamTheisen I added in the doc the kwargs part to be consistent with the rest of the act docs. Also did the changes @mgrover1 suggested.